motoman_driver/MotoPlus/SimpleMessage.h

```diff
@@ -59,10 +59,14 @@ typedef enum
 	ROS_MSG_JOINT_FEEDBACK = 15,
 	ROS_MSG_MOTO_MOTION_CTRL = 2001,
 	ROS_MSG_MOTO_MOTION_REPLY = 2002,
-	ROS_MSG_MOTO_READ_SINGLE_IO = 2003,
-	ROS_MSG_MOTO_READ_SINGLE_IO_REPLY = 2004,
-	ROS_MSG_MOTO_WRITE_SINGLE_IO = 2005,
-	ROS_MSG_MOTO_WRITE_SINGLE_IO_REPLY = 2006,
+	ROS_MSG_MOTO_READ_IO_BIT = 2003,
+	ROS_MSG_MOTO_READ_IO_BIT_REPLY = 2004,
+	ROS_MSG_MOTO_WRITE_IO_BIT = 2005,
+	ROS_MSG_MOTO_WRITE_IO_BIT_REPLY = 2006,
+	ROS_MSG_MOTO_READ_IO_GROUP = 2007,
+	ROS_MSG_MOTO_READ_IO_GROUP_REPLY = 2008,
+	ROS_MSG_MOTO_WRITE_IO_GROUP = 2009,
+	ROS_MSG_MOTO_WRITE_IO_GROUP_REPLY = 2010,

```

Comment:
If we want to do this properly, we'd need to update [REP-I0004](https://github.com/ros-industrial/rep/blob/master/rep-I0004.rst), specifically section [Vendor Specific Messages - Motoman](https://github.com/ros-industrial/rep/blob/master/rep-I0004.rst#motoman).